### DISCUSSION TITLE
[Request] Wireless Redstone CBE receivers forget after relogging

### DISCUSSION THREAD

**Discussion by Vapaman** [ACGaming/UniversalTweaks#315] [Bugfix Suggestions]
https://github.com/TheCBProject/WirelessRedstone/issues/33

How to reproduce:
1) Set Wireless receiver and wireless transmitter to the same channel
2) The connection should work as normal
3) Relog and it stops working

I'm requesting a fix here because Wireless Redstone CBE doesn't seem to be supported anymore

**Comment by thegenghistkhan** [comment]
Same issue for me, except whenever I restart my 1.12.2 Pack server.
